extensions/nba-game-viewer/src/types/standings.types.ts

```diff
@@ -45,9 +45,8 @@ export type Team = {
   losses?: number;
 };
 
-export const Conference = {
-  Eastern: "Eastern",
-  Western: "Western",
-} as const;
-
-export const League = "League" as const;
+export interface Preferences {
+  league: string;
+  conference: string;
+  useLastValue: boolean;
+}
```

Comment:
You don't need this, if you just use `Preferences` already present in _raycast-env.d.ts_

```suggestion
```